lucene/core/src/java/org/apache/lucene/codecs/lucene90/IndexedDISI.java

```diff
@@ -720,6 +805,15 @@ boolean advanceExactWithinBlock(IndexedDISI disi, int target) {
      * return whether this document exists.
      */
     abstract boolean advanceExactWithinBlock(IndexedDISI disi, int target) throws IOException;
+
+    /**
+     * Similar to {@link DocIdSetIterator#intoBitSet}, load docs in this block into a bitset. This
+     * mehtod return true if there are remaining docs (gte upTo) in the block, otherwise false. When
+     * false return, fp of disi#slice is at blockEnd and disi#index is correct but other status vars
+     * are undefined. Caller should decode the header of next block by {@link #readBlockHeader()}.
+     */
+    abstract boolean intoBitsetWithinBlock(
```

Comment:
Nit: for consistency with how it's called on `DocIdSetIterator`.

```suggestion
    abstract boolean intoBitSetWithinBlock(
```